include/openthread/thread_ftd.h

```diff
@@ -219,23 +219,33 @@ uint8_t otThreadGetLocalLeaderWeight(otInstance *aInstance);
 void otThreadSetLocalLeaderWeight(otInstance *aInstance, uint8_t aWeight);
 
 /**
- * Get the Thread Leader Partition Id used when operating in the Leader role.
+ * Get the current Thread Leader Partition Id.
```

Comment:
```suggestion
 * Get the current Thread Network Partition Id.
```
I think it's `Thread Network Partition Id` rather than `Thread Leader Partition Id`, even though the command is `leaderpartitionid`. Thoughts? @librasungirl @zhanglongxia

Reply:
The header file `./include/openthread/thread.h:706` has defined the function `otThreadGetPartitionId()`, so we don't need to define it anymore.